### PR TITLE
ba-76 _handleLogin(userLogin) crashing on page load and logout because userLogin has undefined properties

### PR DESCRIPTION
**Pull request by lyio**
- Introduce null checking for _handleLogin in `<socobo-inventory>`

@flashback2k14  @hetec for review

### CODE REVIEW

File: socobo-inventory.html [hunk 1]
```diff
@@ -78,7 +78,7 @@
         @method _handleLogin
         */
         _handleLogin: function(login) {
-          return login.firebaseUrl + "inventory/" + login.userId + "/";
+          return login ? login.firebaseUrl + "inventory/" + login.userId + "/" : this.location;
```
Comment (flashback2k14) [inline socobo-inventory.html]: Looks good, but what if userId is null? After loggout The userId is null.

Comment (lyio) [inline socobo-inventory.html]: But the login object is not null?

Sebastian Kloppe notifications@github.com schrieb am Fr., 8. Jan. 2016,
15:55:

Comment (flashback2k14) [inline socobo-inventory.html]: nope. after logout the url is "FIREBASE_DB/inventory/null"

**Comment by flashback2k14** [comment]
I.O.

**Pull request merged**